src/main/java/htsjdk/samtools/cram/io/CramLong.java

```diff
@@ -0,0 +1,107 @@
+package htsjdk.samtools.cram.io;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.ByteBuffer;
+
+/**
+ * Methods to read and write CRAM long values as given in the file format specification.
+ */
+
+public class CramLong {
```

Comment:
All methods correspond to those in `CramInt`

Reply:
Looking through `cram_io.h` in htslib, they only seem to use ITF8, LFT8, and int32.

CramInt in htsjdk is only used for Block checksums and Container Header sizes and checksums.  Most other such values are ITF8/LTF8.

So you're right that this does seem unused.  I can pull this change out of this PR into a different branch we can bring back if we find a later use for it.